test/e2e/servicebindingrequest_test.go

```diff
@@ -511,3 +561,53 @@ func serviceBindingRequestTest(
 	_, err = assertDeploymentEnvFrom(todoCtx, f, deploymentNamespacedName, sbrName)
 	require.Error(t, err, "expect deployment not to be carrying envFrom directive")
 }
+
+// CreateServiceBindingRequest implements end-to-end step for creating a Service Binding Request to bind the Backing Service and the Application
+func CreateServiceBindingRequest(
+	todoCtx context.Context,
```

Comment:
When passing the context down the common variable name would be `ctx`, instead of `todoCtx`. Here it goes as an minor suggestion.

Reply:
We don't have this function anymore